### ISSUE TITLE
Research CSV Upload integrateable systems

### ISSUE THREAD

**Issue by katelovescode** [pieforproviders/pieforproviders#2360]
## 📋 Summary

Research and write tickets for CSV upload w/ a 3rd party integration such as AirTable, Periscope, or Retool.

Best option is one that will sync w/ our DB so we can highlight errors/duplicates and keep our DB up to date without building the whole UI for it.

## 🇺🇸 States this change applies to
- [X] ALL
- [ ] Nebraska
- [ ] Illinois

**Comment by katelovescode** [comment]
[WIP] Research notes:
- Periscope is visualization/BI only, no native CSV import or obvious way to get raw data back into our DB
- I've spent a couple of hours trying to get Retool to work with our API and it's close but it's still in progress.  Next steps would be to investigate the attendance upload (they do have native CSV import) and test out column matching
- I also spent a little while doing CSV import in Airtable and it's very user friendly.  I will spend a little time investigating connecting it to our API to automatically import, but I think it's a good alternative.  So far the first downside I've found is that it requires users to make an Airtable account, which is a second account/login they have to manage.  I might be able to work around that, just need some more time to investigate.
